docs/How the library works/walkthrough.md

```diff
@@ -1,287 +0,0 @@
-# Schemaform walkthrough
```

Comment:
This went to https://github.com/usds/us-forms-system/pull/112/files#diff-eb990a3a25de79454a99fcc690947a94R1.